qiskit_addon_sqd/fermion.py

```diff
@@ -593,3 +610,17 @@ def _transition_str_to_bool(string_rep: np.ndarray) -> tuple[np.ndarray, np.ndar
     annihilate = np.logical_or(string_rep == "-", string_rep == "n")
 
     return diag, create, annihilate
+
+def generate_bitstrings(norb, neleca,nelecb, n_orb, n_alpha, n_beta):
```

Comment:
Please write a docstring for this function. If it's meant to be a private function, prefix it with an underscore `_`. Also, please add type annotations to the function arguments.